service/debugger/debugger.go

```diff
@@ -814,11 +884,25 @@ func (d *Debugger) findBreakpoint(id int) []*proc.Breakpoint {
 	return bps
 }
 
+func (d *Debugger) findDisabledBreakpoint(id int) *api.Breakpoint {
+	for _, dbp := range d.disabledBreakpoints {
+		if dbp.ID == id {
+			return dbp
+		}
+	}
+	return nil
+}
+
 // FindBreakpointByName returns the breakpoint specified by 'name'
 func (d *Debugger) FindBreakpointByName(name string) *api.Breakpoint {
 	d.targetMutex.Lock()
 	defer d.targetMutex.Unlock()
-	return d.findBreakpointByName(name)
+
+	bp := d.findBreakpointByName(name)
```

Comment:
I think it's easier to just change `findBreakpointByName` to also search the disabledBreakpoints map and remove `findDisabledBreakpointByName`.